projects/1.16/assets/tips/tips/lang/zh_cn.json

```
@@ -1 +1,72 @@
-{}
\ No newline at end of file
+{
+  "tips.title.tip": "提示",
+  "tips.tip.no_tips": "未加载提示，请检查配置。",
+  "tips.tip.undead_damage": "像骷髅这样的亡灵生物会被治疗药水伤害，会被伤害药水治疗。",
+  "tips.tip.poison_immune": "僵尸、骷髅和蜘蛛免疫剧毒药水。",
+  "tips.tip.witch_conversion": "被闪电击中的村民将变成女巫。",
+  "tips.tip.four_eggs": "抛出鸡蛋时，有极小概率生成4只小鸡。",
+  "tips.tip.nether_lava": "熔岩在下界的流动速度会更快。",
+  "tips.tip.east": "太阳、月亮和星星都向东方移动。",
+  "tips.tip.swamp_slime": "史莱姆会在满月时的沼泽中生成。",
+  "tips.tip.deflect_ghast_fireballs": "你能够通过攻击恶魂发出的火焰弹来改变它的运动方向。",
+  "tips.tip.shulker_armor": "潜影贝在打开时会受到更多的伤害。",
+  "tips.tip.arthropods": "节肢杀手能够对蜘蛛、蠹虫、蜜蜂、末影螨造成额外伤害。",
+  "tips.tip.fossils": "在沼泽和沙漠下能够发现大型化石。",
+  "tips.tip.crouch_blocks": "潜行/蹲下可以防止你从方块边缘掉下来。",
+  "tips.tip.smoke_bees": "在蜂巢下方放置营火会使蜜蜂不容易被激怒。",
+  "tips.tip.lava_fuel": "熔岩桶可以作为熔炉的燃料使用。",
+  "tips.tip.wooden_tool_fuel": "不再使用的木制工具能够为熔炉燃料使用。",
+  "tips.tip.scrap_nuggets": "金制与铁制的工具/盔甲能够冶炼成金/铁粒",
+  "tips.tip.piglin_gold": "猪灵是真的非常喜欢黄金。",
+  "tips.tip.dog_health": "狼的尾巴越低，它的剩余血量就越少。",
+  "tips.tip.dog_rotten_flesh": "你可以通过喂狼吃腐肉来恢复它的血量。",
+  "tips.tip.lure_animals": "你可以在主手/副手拿上动物喜欢食物来引诱动物跟随你。",
+  "tips.tip.cooked_drops": "燃烧死亡的动物有时会掉落熟肉。",
```

```suggestion
  "tips.tip.cooked_drops": "因火焰伤害而死的动物有时会掉落熟食。",
```
原文`Animals killed by fire damage will sometimes drop cooked food.`